.travis.yml

```diff
@@ -48,7 +48,7 @@ matrix:
     - os: linux
       env: TARGET="Linux-X86"      LLDB_TESTS="all"  CLANG="0" COVERAGE="1" PLATFORM="1"
     - os: linux
-      env: TARGET="Linux-X86"      LLDB_TESTS="all"  CLANG="0" RELEASE="1"
+      env: TARGET="Linux-X86"      LLDB_TESTS="all"  CLANG="0" COVERAGE="1"  RELEASE="1"
     - os: linux
       env: TARGET="Linux-X86"      LLDB_TESTS="all"  CLANG="1"
```

Comment:
For some reason, libprofile for i386 clang doesn't want to install. I can do it locally, but travis can't find the package.